src/HogwartsCard.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import './HogwartsCard.css';
+import SortingHatImage from "./images/sortinghat.jpg";
+import App from './App'
+import {
+    BrowserRouter as Router,
+    Route,
+    Link,
+    Switch,
+} from 'react-router-dom';
+
+function HogwartsCard() {
+    return(
+        <React.Fragment>
+            <section id="card">
+                <section id="image">
+                    <img src={SortingHatImage} alt='TheSortingHat'/>
+                </section>
+                <section id="information">
+                    <h1 id="titleCard">The Sorting Hat</h1>
+                    <p id="descriptionCard">The Sorting Hat is one of the cleverest enchanted objects most witches and wizards will ever meet. It literally contains the intelligence of the four founders, can speak (through a rip near its brim) and is skilled at Legilimency, which enables it to look into the wearer’s head and divine his or her capabilities or mood.</p>
+                    <p id="attribution">By J.K. Rowling</p>
```

Comment:
No es incorrecto agregar estilos a los elementos con ids, pero el uso principal de los ids es para diferenciar un elemento único. En este caso cada elemento era único porque no hay más elementos en la app 😅, pero siempre intenta pensarlo en una escala más grande. Si esto fuera una aplicación más grande probablemente vas a querer re-utilizar las reglas de estilos de los botones, de los títulos, del contenido, imagen, etc. Entonces rule-of-thumb "Siempre usar clases, si el elemento es súper único e irrepetible, no hay nadie cómo él, es Michael Jackson, es la última coca del desierto, entonces usar id"